components/generic/ContentHeader.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <header class="row">
+    <b-col
+      cols="12"
+      lg="9"
+      class="col lead mt-3"
+    >
+      <h1
+        data-qa="gallery title"
+      >
+        {{ title }}
+      </h1>
+      <!-- eslint-disable vue/no-v-html -->
+      <div
+        v-if="htmlDescription"
+        v-html="htmlDescription"
+      />
+      <!-- eslint-enable vue/no-v-html -->
+      <p v-if="rawDescription">
+        {{ rawDescription }}
+      </p>
```

Comment:
get rid of this and just pass 'description' in as a prop to simplify the logic here.
Passing plain text through the v-html attribute will still work with plain text.

Then in the various Pages always only pass in 'description'